src/components/table/table.tsx

```diff
@@ -1,19 +1,27 @@
 import './table.css'
 
-import Box from '@mui/material/Box'
+import Stack from '@mui/material/Stack'
 import { default as MuiTable } from '@mui/material/Table'
 import TableBody from '@mui/material/TableBody'
 import TableCell from '@mui/material/TableCell'
 import TableContainer from '@mui/material/TableContainer'
 import TableHead from '@mui/material/TableHead'
+import TablePagination from '@mui/material/TablePagination'
 import TableRow from '@mui/material/TableRow'
 import Typography from '@mui/material/Typography'
-import React from 'react'
+import React, { useState } from 'react'
 
 import { capitalizeFirstLetter } from '../helper'
 import type { TableData, TableHeader } from '../types'
 
+const paginationThreshold = 10
+
 export default function Table(props: TableData) {
+  const [page, setPage] = useState(0)
```

Comment:
Would `currentPage` be a better name?
```suggestion
  const [currentPage, setCurrentPage] = useState(0)
```